esp/logging/loggingagent/cassandraloggingagent/cassandralogagent.cpp

```diff
@@ -193,33 +111,36 @@ void CCassandraLogAgent::ensureKeySpace()
     s.set(NULL);
 }
 
-void CCassandraLogAgent::initTransSeedTable()
+void CCassandraLogAgent::ensureTransSeedTable()
 {
     //Create transaction seed table as needed
     StringBuffer transSeedTableKeys;
     StringArray transSeedTableColumnNames, transSeedTableColumnTypes;
     transSeedTableColumnNames.append("id");
     transSeedTableColumnTypes.append("int");
-    transSeedTableColumnNames.append("agent_name");
-    transSeedTableColumnTypes.append("varchar");
     transSeedTableColumnNames.append("application");
     transSeedTableColumnTypes.append("varchar");
-    transSeedTableColumnNames.append("update_time");
-    transSeedTableColumnTypes.append("timestamp");
-    transSeedTableKeys.set("application, agent_name"); //primary keys
+    transSeedTableKeys.set("application"); //primary keys
 
+    //The defaultDB has transactions table.
     setKeySpace(defaultDB.str());
     cassSession->connect();
-    createTable(defaultDB.str(), "transactions", transSeedTableColumnNames, transSeedTableColumnTypes, transSeedTableKeys.str());
+    createTable(defaultDB.str(), transactionTable.str(), transSeedTableColumnNames, transSeedTableColumnTypes, transSeedTableKeys.str());
 
     unsigned transactionCount = 0;
-    if (executeSimpleSelectStatement("SELECT COUNT(*) FROM transactions", transactionCount) == 0)
+    VStringBuffer st("SELECT COUNT(*) FROM %s;", transactionTable.str());
```

Comment:
Do we actually need the count? if not, we might be better off doing a select (somefield) from transactionTable limit 1;